app/controllers/bookings_controller.rb

```diff
@@ -1,13 +1,35 @@
 class BookingsController < ApplicationController
+  before_action :booking_id, only: %i[destroy]
+
   def new
-    #form
+    @booking = Booking.new
   end
 
   def create
-    # Save the data given by user (params)
+    @user = current_user
```

Comment:
change this to @user = User.find(10) to pretend a user with user_id=10  is logged in